src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,40 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortButton
+  {:abort-submission
```

Comment:
We don't have visibility modifiers for these, unfortunately. So, the convention I've been using is: public methods go first, followed by component lifecycle methods (e.g. :render), followed by private methods. Please move this method after render for consistency with that pattern.